sql/updates/world/2015_01_14_00_world.sql

```diff
@@ -0,0 +1,14 @@
+-- Pandaren Skills
+DELETE FROM `playercreateinfo_skills` WHERE `raceMask` IN (8388608, 16777216, 33554432, 58720256);
+INSERT INTO `playercreateinfo_skills` (`raceMask`, `classMask`, `skill`, `rank`, `comment`)
+VALUES
+	(8388608, 0, 905, 0, 'Language: Pandaren (Neutral)'),
+	(16777216, 0, 905, 0, 'Language: Pandaren (Alliance)'),
+	(33554432, 0, 905, 0, 'Language: Pandaren (Horde)'),
+	(58720256, 0, 899, 0, 'Racial - Pandaren');
+
+-- Language: Common (Alliance)
+UPDATE `playercreateinfo_skills` SET `raceMask` = 18875469 WHERE `skill` = 98;
```

Comment:
Changed this to
```sql
UPDATE `playercreateinfo_skills` SET `raceMask`=`raceMask`|0x1000000 WHERE `skill` = 98;
```

to make it clear that we only want to add a new mask and not change previous ones.